ts/sagas/backendInfo.ts

```diff
@@ -14,12 +14,19 @@ import {
 // load backend info every hour
 const BACKEND_INFO_LOAD_INTERVAL = 60 * 60 * 1000;
 
-// retry loading backend info every 5 seconds on error
-const BACKEND_INFO_RETRY_INTERVAL = 10 * 1000;
+// retry loading backend info every 10 seconds on error
+const BACKEND_INFO_RETRY_INTERVAL = 60 * 60 * 10 * 1000;
 
 export function* backendInfoWatcher(): IterableIterator<Effect> {
   const backendPublicClient = BackendPublicClient(apiUrlPrefix);
-  const getServerInfo = backendPublicClient.getServerInfo;
+
+  function getServerInfo(): Promise<BasicResponseType<ServerInfo> | undefined> {
+    return new Promise((resolve, _) =>
```

Comment:
nit: you can call Promise.resolve(..) or make the method async as a shortcut

Reply:
I actually had to do this at some point since `fetch` may throw an exception - and in TS you can't know if an certain exception can be thrown or not...